graphs/energy/nodes/industry/industry_transformation_generic_coal.ad

```diff
@@ -5,4 +5,4 @@
 - groups = []
 - free_co2_factor = 0.0
 
-~ output.coal = EFFICIENCY(industry_transformation_generic_coal, output, coal)
+~ output.coal = 1.0
```

Comment:
Better to change to `- output.coal = 1.0` and move to line 3

Reply:
Changed it accordingly.